### PR TITLE
Optimize context access on PHP 7.4

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

**Comment by stof** [comment]
is `array_key_exists` optimized by the compiler on PHP 7.4 ? that's a good news

**Comment by fabpot** [comment]
@stof https://github.com/php/php-src/pull/3360

**Comment by stof** [comment]
yeah, that's a great news.

File: CHANGELOG [hunk 1]
```diff
@@ -1,5 +1,6 @@
 * 1.40.0 (2019-XX-XX)
```
Comment (stof) [inline CHANGELOG]: 1.40.0 is already released.

Comment (fabpot) [reply CHANGELOG]: fixed

**Pull request merged**